src/System/TimedScopes/TimedScopeDefinition.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Omex.System.Diagnostics;
+using Microsoft.Omex.System.Logging;
+using Microsoft.Omex.System.Validation;
+using Microsoft.Omex.System.TimedScopes.ReplayEventLogging;
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// Store Timed Scope name and its description
+	/// </summary>
+	public class TimedScopeDefinition
+	{
+		/// <summary>
+		/// Name
+		/// </summary>
+		public string Name { get; }
+
+
+		/// <summary>
+		/// Description
+		/// </summary>
+		public string Description { get; }
+
+
+		/// <summary>
+		/// Description
+		/// </summary>
+		/// <remarks>Could be null</remarks>
+		public string LinkToOCEHandbook { get; }
+
+
+		/// <summary>
+		/// Should the scope be logged only when explicitly demanded
+		/// </summary>
+		public bool OnDemand { get; }
+
+
+		/// <summary>
+		/// Does the scope capture user hashes that are suitable for unique user-based alerting?
+		/// </summary>
+		public bool CapturesUniqueUserHashes { get; }
+
+
+		/// <summary>
+		/// Constructor
+		/// </summary>
+		/// <param name="name">Name</param>
+		/// <param name="description">Description</param>
+		/// <param name="linkToOCEHandbook">Link to OCE Handbook</param>
+		/// <param name="onDemand">Should the scope be logged only when explicitly demanded</param>
+		/// <param name="capturesUniqueUserHashes">Does the scope capture user hashes that are suitable for unique user-based alerting?</param>
+		public TimedScopeDefinition(string name, string description = null, string linkToOCEHandbook = null, bool onDemand = false, bool capturesUniqueUserHashes = false)
```

Comment:
>linkToOCEHandbook [](start = 77, length = 17)

Please expand to On Call Engineer

Reply:
Could you explain more detailed what do you mean by this (should I update the comment)?